src/pages/en/guides/cms/wordpress.md

```diff
@@ -124,62 +123,55 @@ import Layout from '../../layouts/Layout.astro';
 const { slug } = Astro.params;
 
 let res = await fetch(`https://norian.studio/wp-json/wp/v2/dinos?slug=${slug}`)
-let post = await res.json();
+let [post] = await res.json();
 
-// getStaticPaths() is required for static Astro sites
-// if using SSR, you will not need this function
+// The getStaticPaths() is required for static Astro sites.
+// If using SSR, you will not need this function.
 export async function getStaticPaths() {
+  let data = await fetch("https://norian.studio/wp-json/wp/v2/dinos")
+  let posts = await data.json();
 
-  let res = await fetch("https://norian.studio/wp-json/wp/v2/dinos")
-  let posts = await res.json();
-
-	return posts.map((post) => {
-		return {
-			params: { slug: post.slug },
-			props: { post: post },
-		};
-	});
+  return posts.map((post) => ({
+    params: { slug: post.slug },
+    props: { post: post },
+  }));
 }
 ---
-
 <Layout title={post.title.rendered}>
-  <section >
-    <h2 set:html={post.title.rendered}>
-    <article set:html={post.content.rendered} />
-  </section>
+  <article>
+    <h1 set:html={post.title.rendered} />
+    <div set:html={post.content.rendered} />
+  </article>
 </Layout>
-
 ```
 
-
 ### Returning embedded resources
 
 The `_embed` query parameter instructs the server to return related (embedded) resources.
 
 ```astro title="src/pages/dinos/[slug].astro" /&_embed/
-
 ---
 const { slug } = Astro.params;
 
 let res = await fetch(`https://norian.studio/wp-json/wp/v2/dinos?slug=${slug}&_embed`)
-let post = await res.json();
+let [post] = await res.json();
 ---
-
 ```
 
 The `_embedded['wp:featuredmedia']['0'].source_url` property is returned, and can be used to display the featured image on each dinosuar page.
 
 ```astro title="/src/pages/dinos/[slug].astro" {3}
 <Layout title={post.title.rendered}>
-  <section >
+  <article>
     <img src={post._embedded['wp:featuredmedia']['0'].source_url} />
-    <h2 set:html={post.title.rendered}>
-    <article set:html={post.content.rendered} />
-  </section>
+    <h1 set:html={post.title.rendered} />
+    <div set:html={post.content.rendered} />
```

Comment:
I am not sure what the response is but perhaps this could be a good use for our <Fragment> component 
```suggestion
    <Fragment set:html={post.content.rendered} />
```